recipes/motus/meta.yaml

```diff
@@ -0,0 +1,25 @@
+package:
+  name: motus
+  version: "1.0"
+
+source:
+  fn: mOTUs.Linux64bits.tar.gz
+  url: http://www.bork.embl.de/software/mOTU/share/mOTUs.Linux64bits.tar.gz
```

Comment:
Thanks for the help. I changed the build to copy the script to $PREFIX/opt/ with a link to $PREFIX/bin. I also removed the blob from the perl script. I am not checking the binaries on the meta.yaml file because motus tool reference them directly (to the $PREFIX/opt/ folder) and I am not copying them to the bin. However, I am testing if they were generated in the build.sh script.